packages/lib-classifier/src/plugins/drawingTools/models/tools/index.js

```diff
@@ -1,2 +1,4 @@
 export { default as LineTool } from './LineTool'
 export { default as PointTool } from './PointTool'
+export { default as Tool } from './Tool'
+export { default as TranscriptionLineTool } from '@plugins/drawingTools/experimental/models/tools/TranscriptionLineTool'
```

Comment:
Same comment about the import here.

Reply:
Changing this to a relative path broke the export but I'm not sure why.